Code/Framework/AzCore/AzCore/Math/VectorN.h

```diff
@@ -39,6 +39,10 @@ namespace AZ
 
         ~VectorN() = default;
 
+        VectorN& operator=(VectorN&&) = default;
```

Comment:
Don't we have to implement the move assignment manually because we use AZStd::move?

Reply:
No, the move assignment operator would only need to be manually implemented if the default wasn't sufficient.
The default move assignment/move constructor is to invoke move on each of the members.

Reply:
ok, just wanted clarification, because i've seen some PRs in the past that implemented the move constructor but left the normal copy constructor as default.